packages/discord.js-utilities/src/lib/PromptMessage.ts

```diff
@@ -0,0 +1,150 @@
+import type {
+	Message,
+	User,
+	TextChannel,
+	NewsChannel,
+	APIMessageContentResolvable,
+	MessageOptions,
+	MessageAdditions,
+	DMChannel,
+	CollectorFilter,
+	EmojiIdentifierResolvable,
+	MessageReaction
+} from 'discord.js';
+
+/**
+ * This is a [[PromptMessage]], a utility to send a message with Yes/No promt.
+ * You must either use this class directly or extend it.
+ *
+ * [[PromptMessage]] uses actions, these are essentially reaction emojis, when triggered run the said action.
+ * You can utilize your own actions, or you can use the [[PromptMessage.defaultPrompts]].
```

Comment:
This makes absolutely no sense... rewording might be needed as this uses emojis, not "actions", and when clicked, returns the result

Reply:
Please re-review if the new version is better